source/playlunky/playlunky_settings.cpp

```diff
@@ -138,11 +141,13 @@ void PlaylunkySettings::WriteToFile(std::string settings_file) const
             {
                 value = std::string{ known_setting.DefaultValue };
             }
+            /* why did it write overridden settings back to file? sounds like a one-off thing that shouldn't be saved
```

Comment:
I feel like this was in preparation of an in-game settings editor, but it's been so long.